src/models/room-state.js

```diff
@@ -455,7 +455,7 @@ RoomState.prototype.mayTriggerNotifOfType = function(notifLevelKey, userId) {
         powerLevelsEvent &&
         powerLevelsEvent.getContent() &&
         powerLevelsEvent.getContent().notifications &&
-        powerLevelsEvent.getContent().notifications[notifLevelKey]
+        powerLevelsEvent.getContent().notifications[notifLevelKey] !== undefined
```

Comment:
and if it's `null`, `false`, `NaN`, or `[]`? Might be best to change the check to be something like this:

```
!isNaN(powerLevelsEvent.getContent().notifications[notifLevelKey]) 
&& typeof(powerLevelsEvent.getContent().notifications[notifLevelKey]) === 'number'
```

Reply:
I got impatient and switched it and the others in this file over to a utils method `isNumber` I added which does pretty much the above